src/Webhooks.ts

```diff
@@ -79,14 +82,22 @@ export function createWebhooks(
       cryptoProvider: CryptoProvider,
       receivedAt: number
     ): WebhookEvent {
-      this.signature.verifyHeader(
-        payload,
-        header,
-        secret,
-        tolerance || Webhook.DEFAULT_TOLERANCE,
-        cryptoProvider,
-        receivedAt
-      );
+      try {
+        this.signature.verifyHeader(
+          payload,
+          header,
+          secret,
+          tolerance || Webhook.DEFAULT_TOLERANCE,
+          cryptoProvider,
+          receivedAt
+        );
+      } catch (e) {
+        if (e instanceof CryptoProviderOnlySupportsAsyncError) {
+          e.message +=
+            '\nUse `await constructEventAsync(...)` instead of `constructEvent(...).`';
```

Comment:
```suggestion
            '\nUse `await constructEventAsync(...)` instead of `constructEvent(...)`';
```
nit: period inside of \`\`